src/lib/operators/jit_operator/jit_aware_lqp_translator.cpp

```diff
@@ -112,13 +114,13 @@ std::shared_ptr<JitOperatorWrapper> JitAwareLQPTranslator::_try_translate_sub_pl
   // If we can reach the input node without encountering a UnionNode or PredicateNode,
   // there is no need to filter any tuples
   if (filter_node != input_node) {
-    // However, if we do need to filter, we first convert the filter node to a JitExpression ...
-    const auto expression = _try_translate_node_to_jit_expression(filter_node, *read_tuple, input_node);
-    if (!expression) return nullptr;
+    // However, if we do need to filter, we first convert the subplan into a JitExpression
+    const auto jit_predicate_expression = _try_translate_subplan_to_jit_predicate_expression(filter_node, *read_tuples, input_node);
```

Comment:
You don't use "subplan" in the non-JIT translator. Why here?

Reply:
alternatives: 
* "lqp" - nope, we're not turning the *entire* lqp into a JitOperatorWrapper
* "node" certainly not, we're turning *multiple* nodes into a a JitOperatorWrapper

...so "subplan" rose as a term. Not ideal either, as it implies the **entire** subplan gets converted, but imo better than the two other names.

Reply:
ok